### PR TITLE
Add Jones RA Data

### PR DESCRIPTION
**Pull request by borcuttjahns**
Here's code and files for adding the RA tensor data which may be useful for testing the utility of tPLS. The description of the data can be found in this paper:

https://www.nature.com/articles/nchembio.2211

It includes 4-D data containing the cytokine expression of synovial fibroblasts in healthy and rheumatoid arthritis patients in response to stimulant plus inhibitor combinations (Donor x stimulant x inhibitor x cytokine). Disease status is also included in the processing file.

Let me know if there are any changes you would like made or feel free to dive in and change things yourself!

### CODE REVIEW

**Comment by cyrillustan** [comment]
Thank you! We should probably use xarray since there are a lot of labels, and it can also make sure the disease status aligns well with the tensor. I'll look into the paper you posted later.

File: pyproject.toml [hunk 1]
```diff
@@ -10,6 +10,7 @@ python = "^3.8"
 numpy = "^1.22.1"
 pandas = "^1.4.0"
 xarray = "^2022.3.0"
+netCDF4 = "^1.6.0"
```
Comment (cyrillustan) [inline pyproject.toml:13]: What is this package for?

**Review by cyrillustan — APPROVED** [review]
Thank you for putting this together! I made some small changes, but this is all good! Just to clarify, there are only 7 patients? It'll make PLS-DA a bit difficult due to the small data size, but we can see what is appropriate. Also, there is all Rep1 (or Rep 2) is not relevant to one another, right?

**Comment by borcuttjahns** [comment]
Hey @cyrillustan - thanks for modifying this to the repo standards and sorry I didn't see a lot of these comments in real time!

Yes there are only 7 patients, unfortunately. Also I'm not sure what you mean by rep1 and rep2 not being relevant to each other - I believe I take the mean of the two for each patient. You could I suppose not average those in order to increase data size by a factor of 2? Let me know if I'm not understanding the question!

**Comment by cyrillustan** [comment]
@borcuttjahns No worries at all, and thank you for keeping track of this! I tried to ask if Rep1 for all patients was measured at one time/batch and all Rep2 at another. If this is the case (which I suppose is unlikely), maybe we can make Rep1/2 another mode in tensor?
Yes, you did average the data. A small patient size will be challenging for any classification method, and adding replicates may not change that drastically.

**Comment by borcuttjahns** [comment]
@cyrillustan Yes the replicates are from the same patient and should not be different and thus I believe the averaging is the correct way to go about handling this data. Hopefully, there's still something useful that can come out of tPLS using this data!

**Comment by cyrillustan** [comment]
@borcuttjahns I see, so the current handling is correct. We'll certainly try tPLS on this and see what comes out!

**Pull request merged**